templates/default/fulldoc/html/css/style.css

```diff
@@ -82,6 +82,12 @@ body {
   #search { display: none; }
 }
 
+@media (max-width: 320px) {
```

Comment:
We had a requirement to support devices with < 320 px width (like smart phone). I'm not a CSS expert, but the way this is coded in style.css for 920px min-width interferes with 320px unless I copy over the same properties. I effectively am only adding break-word here. I am probably not the best person to untangle / cascade this properly. It removes all horizontal scrolling except for tables and code snippets which is acceptable for the requirement.